### ISSUE TITLE
GTIN

### ISSUE THREAD

**Issue by robertsimcox** [asaquzzaman/woocommerce-to-google-merchant-center#1]
All of our products have been disapproved by Google because they do not have a GTIN.  I've started entering the GTIN as feedUPC as we discussed.  We need the plugin updated to get this to work.  This of course has negatively effected our business!!!

**Comment by robertsimcox** [comment]
Where are you with solving this issue????  Google will be disapproving any item without GTIN today.  If you don't fix your plugin today, we will have no products listed on Google Merchant, which makes your plugin useless!

**Comment by sascha1337** [comment]
robert u fixed it or are you waiting, too ?

**Comment by asaquzzaman** [comment]
Yes, solved this problem. But did not release it for public. Because this plugin can not handle large structure products such as 5000 or more. I am now working with this task. If you have 2000 or less than products you may ask me for update version for GTIN.

**Comment by robertsimcox** [comment]
Technically we have 400 products but basically 15000 variables of all
because they have multiple sizes and colors and GTINs for all of them.

On Fri, Jun 3, 2016 at 1:31 PM, Asaquzzaman notifications@github.com
wrote:

> Yes, solved this problem. But did not release it for public. Because this
> plugin can not handle large structure products such as 5000 or more. I am
> now working with this task. If you have 2000 or less than products you may
> ask me for update version for GTIN.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/asaquzzaman/woocommerce-to-google-merchant-center/issues/1#issuecomment-223642296,
> or mute the thread
> https://github.com/notifications/unsubscribe/ASUbadm6PBL4SXn6Ej7hiLJwtVwLVrasks5qIGTygaJpZM4IYFqx
> .

## 

Corbin's Footwear
559 High Street
Lock Haven PA 17745
570-893-1880

**Comment by asaquzzaman** [comment]
Fixed all problem. Please contact with my mail with your purchase email.

**Comment by robertsimcox** [comment]
It is either corbinsfootwear559@gmail.com or robert_simcox@yahoo.com

On Sat, Jun 4, 2016 at 9:12 PM, Asaquzzaman notifications@github.com
wrote:

> Fixed all problem. Please contact with my mail with your purchase email.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/asaquzzaman/woocommerce-to-google-merchant-center/issues/1#issuecomment-223787244,
> or mute the thread
> https://github.com/notifications/unsubscribe/ASUbaaTfnHueLvxfRAfVo2gQRthiJtNgks5qIiJtgaJpZM4IYFqx
> .

## 

Corbin's Footwear
559 High Street
Lock Haven PA 17745
570-893-1880
